.devcontainer/devcontainer.json

```diff
@@ -2,26 +2,40 @@
   "name": "RStudio-Focal-Amd64",
 
   "build": {
-    "dockerfile": "Dockerfile"
+    "dockerfile": "Dockerfile",
+    "context": ".."
   },
 
   "settings": {
-    "terminal.integrated.shell.linux": "/bin/bash"
+    "terminal.integrated.defaultProfile.linux": "/bin/bash",
+
+    "cmake.configureArgs": ["-DRSTUDIO_TARGET=Server"],
+    "cmake.configureEnvironment": {
+      "GWT_XMX": "-Xmx1000M",
```

Comment:
A lot of this stems from trying to build GWT at the same time as the C++ code. Known issue:  https://github.com/rstudio/rstudio/issues/7660